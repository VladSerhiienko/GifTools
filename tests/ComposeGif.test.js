var wasmBinaryFile = '../bin/GifTools.wasm';
var wasmJSFile = '../bin/GifTools.js';

describe('test add', () => {
    let GifToolsLoader;

    beforeEach(() => {
        GifToolsLoader = require(wasmJSFile);;
    })

    beforeAll(() => {
        jest.useFakeTimers();
    });

    describe('positive integers', () => {
        test('the data is peanut butter', done => {
            try {
                GifToolsLoader({wasmBinaryFile})
                    .then(GifToolsModule => {
                        var result = GifToolsModule._testAdd(49, 55);
                        // expect(result).toBe(100)
                        expect(result).toMatchSnapshot();
                        done();
                    });
            } catch (error) {
                done(error);
            }
        });
    });
});

