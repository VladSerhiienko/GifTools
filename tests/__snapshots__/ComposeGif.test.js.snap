// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test add positive integers the data is peanut butter 1`] = `104`;
